backend/bom-bom-server/src/main/java/me/bombom/api/v1/subscribe/dto/UnsubscribeResponse.java

```diff
@@ -0,0 +1,11 @@
+package me.bombom.api.v1.subscribe.dto;
+
+public record UnsubscribeResponse(
+    String unsubscribeUrl,
+    boolean hasUnsubscribeUrl
+) {
+    public UnsubscribeResponse(String unsubscribeUrl) {
+        this(unsubscribeUrl, unsubscribeUrl != null && !unsubscribeUrl.isBlank());
```

Comment:
저희 기존에 사용하던 방식처럼 이렇게 바꿔줄 수 있을 것 같아요  
```suggestion
        this(unsubscribeUrl, StringUtils.isNotBlank(unsubscribeUrl));
```